package/android/src/main/java/com/mrousavy/camera/CameraView.kt

```diff
@@ -134,68 +137,72 @@ class CameraView(context: Context) :
+            config.video = CameraConfiguration.Output.Enabled.create(
+              CameraConfiguration.Video(
+                pixelFormat,
+                enableFrameProcessor
+              )
             )
-          )
-        } else {
-          config.video = CameraConfiguration.Output.Disabled.create()
-        }
+          } else {
+            config.video = CameraConfiguration.Output.Disabled.create()
+          }
 
-        // Audio
-        if (audio == true) {
-          config.audio = CameraConfiguration.Output.Enabled.create(CameraConfiguration.Audio(Unit))
-        } else {
-          config.audio = CameraConfiguration.Output.Disabled.create()
-        }
+          // Audio
+          if (audio == true) {
+            config.audio = CameraConfiguration.Output.Enabled.create(CameraConfiguration.Audio(Unit))
+          } else {
+            config.audio = CameraConfiguration.Output.Disabled.create()
+          }
 
-        // Code Scanner
-        val codeScanner = codeScannerOptions
-        if (codeScanner != null) {
-          config.codeScanner = CameraConfiguration.Output.Enabled.create(
-            CameraConfiguration.CodeScanner(codeScanner.codeTypes)
-          )
-        } else {
-          config.codeScanner = CameraConfiguration.Output.Disabled.create()
-        }
+          // Code Scanner
+          val codeScanner = codeScannerOptions
+          if (codeScanner != null) {
+            config.codeScanner = CameraConfiguration.Output.Enabled.create(
+              CameraConfiguration.CodeScanner(codeScanner.codeTypes)
+            )
+          } else {
+            config.codeScanner = CameraConfiguration.Output.Disabled.create()
+          }
 
-        // Orientation
-        config.orientation = orientation
+          // Orientation
+          config.orientation = orientation
 
-        // Format
-        val format = format
-        if (format != null) {
-          config.format = CameraDeviceFormat.fromJSValue(format)
-        } else {
-          config.format = null
-        }
+          // Format
+          val format = format
+          if (format != null) {
+            config.format = CameraDeviceFormat.fromJSValue(format)
+          } else {
+            config.format = null
+          }
 
-        // Side-Props
-        config.fps = fps
-        config.enableLowLightBoost = lowLightBoost ?: false
-        config.enableHdr = hdr ?: false
-        config.torch = torch
+          // Side-Props
+          config.fps = fps
+          config.enableLowLightBoost = lowLightBoost ?: false
+          config.enableHdr = hdr ?: false
+          config.torch = torch
 
-        // Zoom
-        config.zoom = zoom
+          // Zoom
+          config.zoom = zoom
 
-        // isActive
-        config.isActive = isActive && isAttachedToWindow
+          // isActive
+          config.isActive = isActive && isAttachedToWindow
+        }
+      } else {
+        cameraSession.close()
```

Comment:
The reason why I don't wanna merge this PR is two things:
1. because we have two paths here that could potentially go out of sync (e.g. it always chooses the `isMounted = false` path but thinks that it updates the config. We don't want branches here, we want one path.
2. Closing a `Closeable` (`CameraSession`) and then re-using it later is an anti pattern. Maybe we wanna do this somehow differently. 

I'll figure something out, but let's leave this PR open for inspiration for now!